scm-ui/ui-webapp/src/containers/NavigationBar.tsx

```diff
@@ -111,13 +125,26 @@ const NavigationBar: FC<Props> = ({ links }) => {
     return () => window.removeEventListener("click", close);
   }, [burgerActive]);
 
+  useEffect(() => {
+    const query = `(max-width: ${devices.desktop.width - 1}px)`;
+    const media = window.matchMedia(query);
+    if (media.matches !== matchQuery) {
```

Comment:
It is not necessary to check the previous value. React does not re render if the same value is set.